### PR TITLE
Prepare release 2020.11

### PR DESCRIPTION
**Pull request by romanstrobl**
Merge of all changes from `develop` into `master` for the `2020.11` release. To be merged after https://github.com/wultra/powerauth-docker/pull/111.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ fi
 
 # Prepare Build Number
 if [ -z ${TAG+x} ]; then
-    export PRODUCT_VERSION="2019.11"
+    export PRODUCT_VERSION="2020.05"
```
Comment (hvge) [inline build.sh:12]: Is this right? The PR title is "Prepare release 2020.11"

**Comment by romanstrobl** [comment]
Please see https://github.com/wultra/powerauth-docker/pull/111 first. It contains changes for this release. This pull request will include those other changes once merged.

**Comment by romanstrobl** [comment]
The pull request is up-to-date now after merge to `develop`.

File: deploy/lib/readme.txt [hunk 2]
```diff
@@ -3,7 +3,7 @@ Add following libraries here:
 - ojdbc6.jar [1]
 - mysql-connector-java.jar [2]
 - postgresql.jar [3]
-- bcprov-jdk15on-164.jar [4]
+- bcprov-jdk15on-165.jar [4]
```
Comment (petrdvorak) [inline deploy/lib/readme.txt:6]: We use 1.67 here

**Review by petrdvorak — APPROVED** [review]
There is one minor typo in the docs, otherwise looks good.

File: deploy/lib/readme.txt (hunk 2 above)
Comment (romanstrobl) [reply deploy/lib/readme.txt]: Fixed, thanks.

**Pull request merged**